libs/gi-localization/assets/locales/en/page_character.json

```diff
@@ -57,5 +57,40 @@
   "tabTeambuff": {
     "team_reso": "Team Resonance",
     "resonance_tip": "The party must be full with 4 characters in order for Elemental Resonance to take effect."
+  },
+  "tabTalent": {
+    "constellation": "Constellation",
+    "constellation_lv": {
```

Comment:
You can use https://www.i18next.com/translation-function/interpolation to minimize the churn in `constellationLvl` and in `talentLvl`